docs/README.md

```diff
@@ -51,7 +51,7 @@ When done, the install script prints out instructions for running and starting
 code-server.
 
 We also have an in-depth [setup and
-configuration](https://coder.com/docs/code-server/v3.11.1/guide) guide.
+configuration](https://coder.com/docs/code-server/v3.11.2/guide) guide.
 
 ## TLS and authentication (beta)
```

Comment:
This URL will break as per https://github.com/cdr/code-server/pull/3953

Reply:
Can't really comment on the correct line - I mean L58

Reply:
Going to do in a follow-up PR since I'm closing this: https://github.com/cdr/code-server/issues/3963